spa/src/components/common/SystemNotification/SystemNotification.tsx

```diff
@@ -0,0 +1,56 @@
+import PropTypes, { InferProps } from 'prop-types';
+import { ReactChild } from 'react';
+import {
+  CheckCircleOutlineOutlined as SuccessIcon,
+  CloseOutlined as CloseIcon,
+  ErrorOutlineOutlined as ErrorIcon,
+  InfoOutlined as InfoIcon,
+  ReportProblemOutlined as WarningIcon,
+  SvgIconComponent
+} from '@material-ui/icons';
+
+import { IconBox, IconBoxIcon, IconButton, Header, Main, SystemNotificationWrapper } from './SystemNotification.styled';
+
+import { SystemNotificationType } from './commonTypes';
+
+const SystemNotificationPropTypes = {
+  type: PropTypes.string.isRequired,
+  header: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  handleClose: PropTypes.func.isRequired
+};
+
+export interface SystemNotificationProps extends InferProps<typeof SystemNotificationPropTypes> {
+  type: SystemNotificationType;
+  children: ReactChild | ReactChild[];
```

Comment:
```suggestion
  children: ReactNode;
```
This is just React + TypeScript trivia.